app/templates/src/main/resources/config/_application-dev.yml

```diff
@@ -1,6 +1,9 @@
 server:
     port: 8080
 
+liquibase:
+    context: development
```

Comment:
Could you use "dev" instead? This is to be consistent with the rest of the generator, where we use the "dev" profile for Maven and Spring